src/Modules/Content/Dnn.PersonaBar.Pages/Pages.Web/src/components/PageDetails/PageUrl/PageExternalUrl/PageExternalUrl.jsx

```diff
@@ -1,4 +1,6 @@
-import React, {Component, PropTypes} from "react";
+import React, {Component} from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
```

Comment:
This is something I assume we should avoid. Is `PageExternalUrl` itself a stateless component? Isn't it an issue that we're making it directly dependent on redux?